airbyte-integrations/connectors/source-facebook-marketing/Dockerfile

```diff
@@ -13,5 +13,5 @@ ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
 
-LABEL io.airbyte.version=0.4.0
```

Comment:
Can you modify this to 0.5.0? Based on semantic versioning checklist [here](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors). This would be considered a minor change.

Reply:
done :heavy_check_mark: